src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__container">
```

Comment:
The element shouldn't be a parent of the block it's related to

Reply:
i don't quite understand. if you mean that it must be named as a main block due to BEM i'll change the class name to .stars-container